nslsii/iocs/thermo_sim.py

```diff
@@ -52,7 +52,7 @@ def t_rbv(T0, setpoint, K, omega, Tvar,):
                     setpoint)
 
         T = t_rbv(T0=self._T0,
-                  **{k: getattr(self, k).value
```

Comment:
This is inside of a caproto IOC and I don't think will work!

Reply:
Reverted this change. Thanks for catching it up!